config/default.js

```diff
@@ -312,4 +313,10 @@ module.exports = {
   enableAMInstallButton: false,
 
   enableInlineAddonReview: false,
+
+  // The withExperiment HOC relies on this config to enable/disable A/B
+  // experiments.
+  experiments: {
+    home_hero: false,
```

Comment:
do we want this to be false? or do we want to make sure to set this on for stage and prod? I guess I was under the impression this was supposed to go live this week?

Reply:
There is no rush and no deadline. This will go live next week I suppose.